src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/utils/VisitBalancesUtilTest.kt

```diff
@@ -0,0 +1,124 @@
+package uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.utils
+
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Test
+import uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.dto.prison.api.VisitBalancesDto
+import java.time.LocalDate
+
+class VisitBalancesUtilTest {
+
+  @Test
+  fun `test available VOs is a total of VO and PVO`() {
+    // Given
+    val visitBalance = VisitBalancesDto(3, 4, null, null)
```

Comment:
1 more suggestion:

Could we use named parameters in test object creation, to show whats being populated more easily ?

Reply:
Updated.